P2P_chat/P2P_chat/main.cpp

```diff
@@ -0,0 +1,123 @@
+#include <iostream>
+#include <winsock.h>
+
+#include "global.h"
+#include "server.h"
+#include "client.h"
+
+
+
+void getUserChoice(char&);
+void getServerAddress(char[], int&);
+
+
+
+int main(int argc, char argv[]) {
+
+    using namespace p2p_chat;
+    using namespace p2p_chat::global;
+
+
+    char userChoice;
+    char kbInput[MAX_USERNAME_LENGTH];
+
+
+    std::cout << "Choose option by inserting its number:\n1) Start session,\n2) Connect to session.\n> ";
```

Comment:
Can you remove this branching for extra mark? Start both clients in the same way: create listening socket and wait for incoming connection. Messaging logic seems to be the same. No 'server' option is required here.